openfisca_france/parameters/taxation_capital/impot_fortune_immobiliere_ifi_partir_2018/reduc_impot/plafond_somme_trois_reductions_pme_fcip_fip_pme_dons.yaml

```
@@ -6,7 +6,7 @@ values:
     value: 50000
 metadata:
   short_label: Plafond pour la somme des trois réductions (PME, FCPI/FIP sur PME et dons)
-  last_value_still_valid_on: "2023-03-31"
+  last_value_still_valid_on: "2024-04-26"
```

La date à indiquer est celle à laquelle le pramètre a été vérifié pour la dernière fois, donc plutôt la date du jour.

A moins que ce ne soit volontaire car la date vient de l'harmonisation et n'a pas été vérifiée ?

```suggestion
  last_value_still_valid_on: "2025-12-18"
```